index.html

```diff
@@ -5081,9 +5031,60 @@ <h2 id="AudioWorkletNode">
             connected with other built-in <a><code>AudioNode</code></a>s to
             form an audio graph.
           </p>
-          <dl title=
-          "[Constructor(BaseAudioContext context, DOMString name, optional AudioWorkletNodeOptions options)] interface AudioWorkletNode : AudioNode"
-          class="idl">
+          <dl title="interface AudioWorkletNode : AudioNode" class="idl">
+            <dt>
+              Constructor(BaseAudioContext context, DOMString name, optional
+              AudioWorkletNodeOptions options)
+            </dt>
+            <dd>
+              <p>
```

Comment:
I believe the constructor section should describe what should happen in the construction procedure. Currently we have something like this in the spec: https://webaudio.github.io/web-audio-api/#constructors.